packages/file_selector/file_selector_web/example/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: file_selector_web_integration_tests
+publish_to: none
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  build_runner: ^1.10.0
+  file_selector_web:
+    path: ../
+  flutter_driver:
+    sdk: flutter
+  flutter_test:
+    sdk: flutter
+  integration_test:
+    sdk: flutter
+
+environment:
+  sdk: ">=2.12.0-0 <3.0.0"
+  flutter: ">=1.26.0-0" # For integration_test from sdk
```

Comment:
I thought null safety support was post-1.26?

Reply:
Yeah, I changed this to `1.27.0-0`. I think this slipped through the cracks in url_launcher_web, though. I might revisit that one after everything is migrated.